backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/importer/lbe/types/LbeData.kt

```diff
@@ -0,0 +1,18 @@
+package xyz.elitese.ehrenamtskarte.importer.lbe.types
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.annotation.JsonRootName
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlCData
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement
+
+// Please note that you must not use a data class because there is a issue with the json property
+// on data classes in kotlin --> mindfuck
```

Comment:
Do we use the kotlin module for jackson? https://github.com/FasterXML/jackson-module-kotlin
It seems it should be supported :thinking: